backend/README.md

```diff
@@ -48,3 +48,13 @@ NOTE: priority of configurations is following: command line arguments, environme
 To run tests you need to have [`pg_tmp`](http://eradman.com/ephemeralpg/) to run DB for tests.
 Then run `make test` to execute tests.
 This command automatically executes [`poetry`](https://python-poetry.org/) to obtain Python dependencies.
+
+## Generate sample data
+
+Apart from `edna-server` executable we provide `edna-generator`.
+This tool can generate sample data for Edna and put it into DB.
+You can build and run it the same way as `edna-server` (via `stack` or `cabal`).
+You can also run it using [`Makefile`](./Makefile).
+It has the same options as `edna-server` plus some additional options that specify how much data to generate.
```

Comment:
Can you add comment please, that to see all available options we can call `edna-generator --help`?

Reply:
Done.